src/callbacks_step/analysis.jl

```diff
@@ -208,7 +207,8 @@ function (analysis_callback::AnalysisCallback)(integrator)
     # Calculate current time derivative (needed for semidiscrete entropy time derivative, residual, etc.)
     du_ode = first(get_tmp_cache(integrator))
     @notimeit timer() rhs!(du_ode, integrator.u, semi, t)
-    du = wrap_array(du_ode, mesh, equations, solver, cache)
+    u  = wrap_array_plain(integrator.u, mesh, equations, solver, cache)
```

Comment:
Let's see the effect in af60aa8c

Reply:
Sorry, I fail to see how 4adf208 undoes this, since it only acts on `analysis_dg[123]d.jl` as far as I can tell. Also, it reintroduces `StrideArray` for the caches again - was that intentional?

Reply:
Sorry, wrong commit ID above - I meant af60aa8 instead of 4adf208